x-pack/plugins/observability_solution/apm/public/components/app/service_map/index.tsx

```diff
@@ -52,6 +53,10 @@ function LoadingSpinner() {
   return <EuiLoadingSpinner size="xl" style={{ position: 'absolute', top: '50%', left: '50%' }} />;
 }
 
+function wrapReturn(content: JSX.Element) {
```

Comment:
nit: `wrapReturn` sounds too generic. how about this?

```suggestion
function wrapServiceTab(content: JSX.Element) {
```